src/components/VSelect/VSelect.js

```diff
@@ -176,7 +176,7 @@ export default {
 
       let selectedItems = this.computedItems.filter(i => {
         if (!this.isMultiple) {
-          return this.getValue(i) === this.getValue(val)
+          return this.valueComparator(this.getValue(i), val)
```

Comment:
Is there a reason for not parsing val through `getValue`? This is how it is done in the other methods.

Reply:
Yes, because `val`'s default value is `inputValue`, and `getValue` is used to get the value from an item.
Example:
```js
// val = this.inputValue (assuming is not multiple)
{x: 1, y: 2}

// items
[
  // item
   {
       label: 'Item 1',
       value: {x: 1, y: 2} // taken by getValue() 
   },
   // ... other items
]
```